authors/apps/authentication/serializers.py

```diff
@@ -110,6 +107,12 @@ def validate(self, data):
                 'This user has been deactivated.'
             )
 
+        #If user has not verified their email account an exception will be raised
+        if not user.is_verified:
```

Comment:
Let's allow user to login even if they are not verified

Reply:
Done